Userland/Libraries/LibJS/Runtime/Map.cpp

```diff
@@ -86,6 +86,9 @@ void Map::visit_edges(Cell::Visitor& visitor)
         visitor.visit(value.key);
         visitor.visit(value.value);
     }
+    // NOTE: The entries in m_keys are already visited by the walk over m_entries above.
+    //       The (void) cast here is to suppress a warning from LibJSGCVerifier.
+    (void)m_keys;
```

Comment:
Does `visitor.ignore(m_keys)` also suppress the warning?

Reply:
Oh duh, yes that would be much nicer!